tests/console/kubeadm.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test kubeadm installation and bootstrap single k8s cluster
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use strict;
+use base "consoletest";
+use testapi;
+use caasp;
+
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $packages = 'kubernetes-client kubernetes-kubelet kubernetes-kubeadm docker-kubic cri-tools';
+    trup_install($packages);
+
+    record_info 'Test #2', 'Test: Start the services';
+    assert_script_run('systemctl enable --now docker');
```

Comment:
it is up to you to decide if it is important here or not , just FYI if systemctl fail to start service you will not always getting non-zero error codes ( you can get service in state "starting" which would be fine from systemctl perspective but not fine for you). So I recommend you to add here at least call  `systemctl status dcoker` or even better  `systemctl status dcoker | grep -q 'running'`

Reply:
@okurz according to systemctl documentation  usage of "--now" will extend behavior of enable to enable-and-start